src/os_common/os_fstream_stdio.c

```diff
@@ -50,7 +50,7 @@ __stdio_printf(WT_SESSION_IMPL *session, WT_FSTREAM *fs, const char *fmt, va_lis
 {
```

Comment:
Please remove the session.

Reply:
The session is used in other implementations of the `fstr_printf` function so I've just added a WT_UNUSED instead.